src/tests/integration/test-data/lemmas.k

```diff
@@ -60,8 +60,9 @@ module SUM-TO-N-INVARIANT
    andBool #rangeUInt(256, S)
    andBool GAS_AMT >=Int N *Int 178
    andBool CONDITION ==K bool2Word ( N:Int ==Int 0 )
-   andBool PROGRAM ==K #binRuntime(S2KtestZModLoopsTest)
```

Comment:
Does `#binRuntime` no longer exist, why has this change happened?

Reply:
I mentioned this in my [previous comment](https://github.com/runtimeverification/kontrol/pull/594/files#r1623165796).